tests/test_cirq_device.py

```diff
@@ -410,25 +363,3 @@ def test_apply_two_wires(self, cirq_device_2_wires, gate, expected_cirq_gates):
 
         for i in range(len(ops)):
             assert ops[i].gate == expected_cirq_gates[i]
-
-    # @pytest.mark.parametrize("operation,par,expected_cirq_gates", [
-    #     ("BasisState", [[0]]),
-    #     ("BasisState", [[1]]),
-    #     ("QubitStateVector", [[1, 0]]),
-    #     ("QubitStateVector", [[0, 1]]),
-    #     ("QubitStateVector", [[1/math.sqrt(2), 1j/math.sqrt(2)]]),
-    #     ("QubitStateVector", [[1/2, math.sqrt(3)/2]]),
-    # ])
-    # def test_apply_state_preparations_single_wire(self, cirq_device_1_wire, operation, par, expected_cirq_gates):
-    #     """Tests that apply adds the correct gates to the circuit for state preparations."""
```

Comment:
I wonder why this was commented out

Reply:
I'm guessing it's because Cirq treats state initializations differently than other ops (provided as an input to `simulate` or `run`, rather than as an explicit op).